.github/workflows/l4lb.yaml

```diff
@@ -0,0 +1,63 @@
+name: Cilium L4LB XDP
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  issue_comment:
+    types:
+      - created
+  # Run every 6 hours
+  schedule:
+    - cron:  '0 2/6 * * *'
```

Comment:
I noticed that [the schedule for other workflows](https://github.com/cilium/cilium/search?q=%22run+every+6+hours%22) is always off by one, such that no two workflows run at the same time. I'm not sure what's the rationale for that (cc @michi-covalent, cf. https://github.com/cilium/cilium/commit/3b0d4fbde75340a57273f1e79f661fcaf5df5c81).

Reply:
Made it off by one (`0 4/6 * * *`).

Reply:
> I'm not sure what's the rationale for that

Maybe to reduce a likelihood to hit the concurrent runners per account quota .

Reply:
Ah yes. That could be a good reason :+1: